compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CompilerPluginRunner.java

```diff
@@ -123,14 +125,14 @@ public void visit(BLangPackage pkgNode) {
             return;
         }
 
-        pluginList.forEach(plugin -> plugin.process(pkgNode));
```

Comment:
As we discussed offline, shall we keep this logic as it is and introduce a separate visit for BLangTestablePkg. ?

Reply:
Added a separate visit with 5eb96fa